modules/ROOT/pages/create-instance-task-flex.adoc

```diff
@@ -24,18 +24,141 @@ See xref:gateway::flex-install.adoc[Install Flex Gateway] and xref:gateway::flex
+** *Single Upstream Service* +
+For Flex Gateway versions 1.4.x or earlier or Flex Gateway version 1.5.x with a single upstream service, configure the following upstream configuration settings:
 +
 include::partial$api-configuration-tables.adoc[tags=flex-upstream]
+** *Multiple Upstream Services* +
+Flex Gateway version 1.5.x or later supports API instances that expose multiple upstream services through a single consumer endpoint. +
+To configure multiple upstream services, see <<traffic-management>>.
 . Click *Next*.
 +
 include::partial$task-configure-proxy.adoc[tags=last-steps]
 
 
+[[traffic-management]]
+== Multiple Upstreams for Flex Gateway Running in Connected Mode
+
+Flex Gateway version 1.5.x or later supports API instances that expose multiple upstream services through a single consumer endpoint.
+
+Flex Gateway manages request traffic by using different routes that can each direct traffic to multiple upstream services. Flex Gateway directs traffic to the routes using the route order the individual route's rules.   
+
+In the following diagram, different routes manage requests to flight information databases and to a customer service application. Route one has two upstream services defined, which direct 70% of requests to a stable database and 30% of requests to a beta database.  
+
+image:multiple-upstreams.png[Flex Gateway manages the traffic to multiple upstreams]
+
+=== Routes
+
+Each API instance supports up to 10 routes and each route can support up to 10 upstream services. Configure what requests a route can receive by defining route rules and a route order. At least one router per API instance is required. 
+
+Before adding additional routes, enter an optional *Route label* for clarity.
+
+You can add additional routes by clicking *Add Route*, and you can delete routes by clicking the *Trash Can* icon (image:trash-can-icon.png[2%,2%]). If only one route is defined, you cannot delete that route.
+
+
+==== Route Rules
+
+You can direct requests to different routes by using route rules. 
+
+To view and edit route rules, click *Route Rules*.
+
+All rules are optional. Not defining a value for a rule means that aspect of the request is not considered. For example, not specifying a host means that the route can service any host if the request meets the other route rules. Not defining any rule means that the route can service every request.
+
+Different routes can support the same upstream services. If you cannot capture all requests for a particular set of upstream services in a single route rule set, you can define multiple routes with different rules for full coverage.
+
+Only requests that meet all of the rules defined in the ruleset are directed to that route.
+
+Flex Gateway supports the following route rules:
+
+Method::::
++
+Defines the types of request methods that the route can service.
++
+You can select multiple methods for each route. Only requests that are one of the defined methods are directed to this route.
```

Comment:
Are requests a method?

Reply:
Yes, you could have GET requests or POST requests and so on.

Reply:
Ah - REST requests